designdocs/provider_permissions.md

```diff
@@ -0,0 +1,207 @@
+# Delegator <-> Provider Grants
+
+## Context and Scope
```

Comment:
I don’t think this should be in this PR

Reply:
it shouldn't be, but these changes show up as merge conflicts when I try to merge my branch (I haven't made any changes to this file other than merging main into my branch